src/reducers/vms.js

```diff
@@ -59,10 +52,8 @@ function vms (state, action) {
   logDebug(`The 'vms' reducer action=${JSON.stringify(hidePassword({ action }))}`)
 
   switch (action.type) {
-    case 'UPDATE_VM':
+    case 'UPDATE_VMS':
```

Comment:
Good point, accidentally moved to UserPortal.
Moving back, to be collocated with the reducer